mep3_behavior_tree/CMakeLists.txt

```diff
@@ -22,7 +22,7 @@ find_package(ament_cmake REQUIRED)
 # uncomment the following section in order to fill in
 # further dependencies manually.
 # find_package(<dependency> REQUIRED)
-find_package(BehaviorTreeV3 REQUIRED)
+find_package(behaviortree_cpp_v3 REQUIRED)
```

Comment:
Nisam siguran zasto, ali u ROS indeksu paketa [`behaviortree_cpp_v3`](https://index.ros.org/p/behaviortree_cpp_v3/) vraca nula dostupnih verzija za bilo koju iteraciju ROS2 🧐

Reply:
Srećom i dalje prepoznaje [behaviortree_cpp_v3](https://github.com/BehaviorTree/BehaviorTree.CPP/tree/v3.8#how-to-compile-plain-old-cmake).